portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.data;
```

Comment:
Good job on creating separate files for each class - this makes it more readable.